test/test.js

```diff
@@ -76,6 +76,19 @@ describe('cacheman-file', function() {
     });
   });
 
+  it('should sanitize key', function(done) {
```

Comment:
I agree on that first part: the test just checks for the functionality.
However, I'm not completely sure if the test (and the module in question) should go that far; as far as concerned, the developer using the API is agnostic about _how_ the cache works and just expect it to deliver the functionality described.
Bear in mind that this same test wouldn't have passed with the original code; maybe the description of the test is not accurate? (more like: `'it should allow special characters'` or something like that?).
Let me know your thoughts. Thanks for reviewing!